sources/HUBComponentWrapper.h

```diff
@@ -167,6 +167,9 @@ willUpdateSelectionState:(HUBComponentSelectionState)selectionState;
 /// Whether the wrapped component's view has appeared since the model was last changed
 @property (nonatomic, readonly) BOOL viewHasAppearedSinceLastModelChange;
 
+/// Whether the wrapped component's view has appeared for the first time
+@property (nonatomic, assign, nullable, readonly) NSNumber *viewHasAppearedForTheFirstTime;
```

Comment:
How about using an integer to keep track of the apperance count instead? More explicit than relying on `nil` semantics (IMO).